content/derisking/index.html

```diff
@@ -0,0 +1,95 @@
+        <div class="usa-media-block__body measure-5">
+          <h2 class="">
+            Only 13% of large government IT projects&nbsp;succeed<sup>*</sup>
+          </h2>
+          <p>Implementing custom software projects can be extraordinarily costly and risky in a government setting. Waterfall software development remains standard at all levels of government. Outdated budgeting and oversight processes have been designed around these very waterfall processes. Moving toward a user-centered, agile approach to this work will save millions of dollars in spending on bad software.</p>
+          <p class="font-body-2xs text-base-dark">
+            <span class="text-primary"> * </span> From The Standish Group’s &ldquo;Haze&rdquo; report
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="usa-graphic-list usa-section">
+  <div class="grid-container">
+    <hr>
+    <h2 id="guide-teasers"> Field Guides </h2>
+
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col">
+        <a href="#/TODO/">
+          {% image_with_class "assets/derisking/img/capitol-03-124.png" "" "" %}
+          <h3 class="font-heading-lg margin-top-1 margin-bottom-0 usa-link">
+            State Software Budgeting Handbook
+          </h3>
+        </a>
+          <p>60-page guide for executives, budget specialists, legislators, and other decision makers who fund or oversee&nbsp;state government technology projects.</p>
+      </div>
+
+      <div class="tablet:grid-col">
+        <a href="#/TODO/">
+          {% image_with_class "assets/derisking/img/shield-02-124.png" "" "" %}
+          <h3 class="font-heading-lg margin-top-1 margin-bottom-0 usa-link">
+            Federal Field Guide
+          </h3>
+        </a>
+          <p>128-page guide for federal executives, product owners, budget specialists and agency staff who fund or oversee federal government technology projects.</p>
+      </div>
+
+      
+    </div>
+  </div>
+</section>
+
+<section class="usa-graphic-list usa-section usa-prose bg-accent-cool-lightest">
+  <div class="grid-container">
+    <div class="usa-graphic-list__row grid-row grid-gap">
+      <div class="usa-media-block">
+        {% image_with_class "assets/derisking/img/workshop-01-124.png" "usa-media-block__img" "" %}
+        <div class="usa-media-block__body measure-5">
+          <h2 id="workshops">Workshops</h2>
+          <p>18F can jump start your software development project with workshops that are tailored to reduce project risk, meet your goals, and deliver results to your users. These workshops are designed to teach the basics of modern software development best practices, coach your team through a shared understanding of the problem and likely solution, and deliver actionable next steps, such as a draft solicitation that is ready for release.
+          </p>
+
+          <a class="usa-button usa-button" href="https://18f.gsa.gov/contact/">Contact us</a>
```

Comment:
Just observing that the original derisking guide uses the `usa-button--secondary` class here.

Reply:
Well spotted, I made this visual change because the active state of this style of button blends into the background.